simapp/genesis.go

```diff
@@ -13,9 +12,9 @@ import (
 // Within this application default genesis information is retrieved from
 // the ModuleBasicManager which populates json from each BasicModule
 // object provided to it during init.
-type GenesisState map[string]json.RawMessage
+type GenesisState map[string]proto.Message
```

Comment:
Where is this type used? It seems we use `map[string]proto.Message` everywhere anyways, maybe is it fine to delete this type?

Reply:
Agree, I think it doesn't have any methods defined, so we can just remove it